[2021-08-04 14:05:05,076] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_C 2021-08-04T14:04:54.460692+00:00 [queued]>
[2021-08-04 14:05:05,102] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_C 2021-08-04T14:04:54.460692+00:00 [queued]>
[2021-08-04 14:05:05,102] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-04 14:05:05,102] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-08-04 14:05:05,102] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-04 14:05:05,115] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): training_model_C> on 2021-08-04T14:04:54.460692+00:00
[2021-08-04 14:05:05,121] {standard_task_runner.py:52} INFO - Started process 2203 to run task
[2021-08-04 14:05:05,127] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'training_model_C', '2021-08-04T14:04:54.460692+00:00', '--job-id', '2515', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmp6ggw941k', '--error-file', '/tmp/tmp5a8jqee3']
[2021-08-04 14:05:05,128] {standard_task_runner.py:77} INFO - Job 2515: Subtask training_model_C
[2021-08-04 14:05:05,178] {logging_mixin.py:104} INFO - Running <TaskInstance: my_dag.training_model_C 2021-08-04T14:04:54.460692+00:00 [running]> on host 04479ea62028
[2021-08-04 14:05:05,240] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=training_model_C
AIRFLOW_CTX_EXECUTION_DATE=2021-08-04T14:04:54.460692+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-04T14:04:54.460692+00:00
[2021-08-04 14:05:05,241] {python.py:151} INFO - Done. Returned value was: 1
[2021-08-04 14:05:05,272] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=training_model_C, execution_date=20210804T140454, start_date=20210804T140505, end_date=20210804T140505
[2021-08-04 14:05:05,311] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-08-04 14:05:05,341] {local_task_job.py:149} INFO - Task exited with return code 0
